EIPS/eip-7939.md

```diff
@@ -30,6 +30,8 @@ It is a basic building block used in math operations, byte operations, compressi
 - bitmaps (for finding the next/previous set/unset bit)
 - post quantum signature schemes
 
+Adding a `clz` opcode will also lead to cheaper ZK proving costs. The fastest known Solidity implementation uses several dynamic bitwise right shifts `shr`, which are very expensive to prove. In SP1 rv32im, a 32-bit right shift costs 1.6x more than a 32-bit mul.
```

Comment:
I think instead of "fastest" the "cheapest" is meant here, right?

For readers not familiar in the subject I would add a quick note what SP1 (zkVM) and rv32im (processor) is for clarity to users new to this subject